eudemo/eudemo/ivc/divertor_silhouette.py

```diff
@@ -420,6 +420,23 @@ def _get_wire_end_with_largest(wire: BluemiraWire, axis: str) -> np.ndarray:
         """
         return DivertorSilhouetteDesigner._get_wire_end(wire, axis, operator.gt)
 
+    def _get_wire_ends_by_psi(self, wire: BluemiraWire) -> Tuple[np.ndarray, np.ndarray]:
+        """
+        Get the coordinates of the ends of a wire where the end
+        with higher psi is returned first
+        """
+        start_point = wire.start_point()
+        end_point = wire.end_point()
+
+        psi_start = self.equilibrium.psi(x=start_point.x[0], z=start_point.z[0])
+        psi_end = self.equilibrium.psi(x=end_point.x[0], z=end_point.z[0])
+
+        # todo, should be the opposite but I don't know how to use psi_norm
+        # and the value at specific points
+        if psi_start < psi_end:
```

Comment:
I have a feeling this could the bug @OceanNuclear found in #3037 wrt to start and end points. Fix is coming in #3095 just trying to make the test catch it